### PR TITLE
add example of s3 secrets

### PR DESCRIPTION
**Pull request by kevinawoo**

### CODE REVIEW

**Comment by kevinawoo** [comment]
i'm fixing the builds in another pr

**Comment by mikerowehl** [comment]
bin/secrets is whitelisted into the S3 configuration. Do we want to tell folks to put their secrets script into their config repo? Cause we pull S3 config and then run secrets, I would think that might be confusing. Shouldn't we instead maybe put some notes in about creating a bin/secrets file using the configurator?

File: deb-config/spinnaker/bin/secrets [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+
+
+# This is an example of using a private s3 bucket.
+aws s3 cp s3://secrets-bucket/spinnaker/production/dockerhub.password /opt/spinnaker/secrets/
+aws s3 cp s3://secrets-bucket/spinnaker/production/kubernetesConfig /opt/spinnaker/secrets/
+
+
+# Example of jenkins username and password. -local.yml will read variables from /opt/spinnaker/env/resolved.env
+aws s3 cp s3://secrets-bucket/spinnaker/production/jenkins.credentials /opt/spinnaker/secrets/
+cat /opt/spinnaker/secrets/jenkins.credentials >> /opt/spinnaker/resolved.env
```
Comment (mikerowehl) [inline deb-config/spinnaker/bin/secrets]: The dest path for the aggregated resolved.env is wrong. But otherwise looks good to me, ready to merge

**Comment by kevinawoo** [comment]
merged with @mikerowehl approval in comments

**Pull request merged**